src/redis-cli.c

```diff
@@ -1876,6 +1904,8 @@ static void usage(void) {
 "  --scan             List all keys using the SCAN command.\n"
 "  --pattern <pat>    Keys pattern when using the --scan, --bigkeys or --hotkeys\n"
 "                     options (default: *).\n"
+"  --quoted-pattern <pat> Same as --pattern, but the specified string can be\n"
+"                         quoted, in order to pass an otherwise non binary-safe string.\n"
```

Comment:
maybe in both cases (this help line and the one above), it would help to give an example like `"bla \0\n\xff"`